scripts/elasticsearch.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# Install Elasticsearch in /tmp and run it.
+set -ev
+mkdir /tmp/elasticsearch
+wget -O - https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-6.2.4.tar.gz | tar xz --directory=/tmp/elasticsearch --strip-components=1
+/tmp/elasticsearch/bin/elasticsearch-plugin install analysis-icu
+/tmp/elasticsearch/bin/elasticsearch -E http.port=9201 -d
```

Comment:
I've added `-E path.data=/tmp/elasticsearch` to Es 6.2, and also changed the Es 1.6 one so that it uses a _different_ data path, just to be sure. [There's also `path.logs`](https://www.elastic.co/guide/en/elasticsearch/reference/master/path-settings.html) but I haven't seem anything to suggest that we need to change that (not even sure that we actually need to change `path.data`). Will leave this as it is now unless we get problems.